### PR TITLE
[ruby] Add 1.x support in Ruby weblogs

### PR DESCRIPTION
**Pull request by vpellan**
## Motivation

I wanted to enable easy wins, but also wanted to add the version since it works in the manifest. Trying to run system-tests on 1.x branch was not working. 1.x branch is still active and should still be tested against system-tests

## Changes

Both auto instrumentation and healthcheck endpoints were using `Datadog` namespace, but 1.x is using `DDTrace`. I applied what has been done in Sinatra weblogs and parametric weblog for auto-instrumentation (require in begin/rescue block), and added a condition for getting the version in healthcheck endpoints (`defined?(Datadog::VERSION)`)

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by Strech — APPROVED** [review]
👍🏼

**Review by cbeauchesne — APPROVED** [review]

File: utils/build/docker/ruby/rack/config.ru [hunk 1]
```diff
@@ -7,7 +7,15 @@ require 'json'
...
-require 'datadog/auto_instrument'
+begin
+  require 'ddtrace/auto_instrument'
+rescue LoadError
+end
+
+begin
+  require 'datadog/auto_instrument'
+rescue LoadError
+end
```
Comment (y9v) [inline utils/build/docker/ruby/rack/config.ru:18]: I think we should unify those:

```ruby
begin
  require 'datadog/auto_instrument'
rescue LoadError
  require 'ddtrace/auto_instrument'
end
```

This was we know that at least one of them was loaded (or another `LoadError` will be raised)

**Review by y9v — CHANGES_REQUESTED** [review]

File: utils/build/docker/ruby/rack/config.ru (hunk 1 above)
Comment (vpellan) [reply utils/build/docker/ruby/rack/config.ru:18]: Thanks, I'll do that in Sinatra and parametric weblogs too then

Comment (vpellan) [reply utils/build/docker/ruby/rack/config.ru]: Fixed in 801bd8ab996e8465a2fe20ddcebaebc55a15595c

**Review by y9v — APPROVED** [review]

**Pull request merged**